tachyon/zk/plonk/permutation/grand_product_argument.h

```diff
@@ -44,7 +44,7 @@ class GrandProductArgument {
 
     // NOTE(chokobole): It's safe to downcast because domain is already checked.
     RowIndex size = static_cast<RowIndex>(prover->pcs().N());
-    std::vector<F> z(size + 1);
+    std::vector<F> z(size + 1, F::Zero());
```

Comment:
Do we need to initialize this? It seems like `denominator_callback` fills the values.